csharp/src/Microsoft.ML.OnnxRuntime/OrtAllocator.cs

```diff
@@ -27,6 +27,50 @@ public enum OrtMemType
         Default = 0,                        // the default allocator for execution provider
     }
 
+    /// <summary>
+    /// This class encapsulates an arena configuration information that will be used to define the behavior
+    /// of an arena based allocator
```

Comment:
> of an arena based allocator [](start = 7, length = 28)

May be a few words what is arena based allocator

Reply:
I think @pranavsharma is going to add documentation regarding these parameters in the perf tuning md file. He might have to give a brief intro to arena based allocators there, I prefer to just point to that (which I have) and have ocumentation centralized rather than strewn all over.